app/build.gradle

```diff
@@ -15,6 +15,19 @@ android {
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
 
+    productFlavors {
+        elimu {
+            // Default app collection (elimu.ai)
+            dimension 'mode'
+        }
+        custom {
+            // Custom app collection (custom project)
+            applicationId "ai.elimu.appstore_custom"
```

Comment:
@bvalbo Note that the Appstore for your project will get a new package name

Reply:
Noted. For the future - may we change icon and possibly the pck name so it's associated a bit more with Leap Learning?